General

EmanueleC opened the pull request:
**Reference to the issue (if any)**: #15 

**Description**

EmanueleC committed:
add some definitions

EmanueleC committed:
add some definitions

EmanueleC committed:
add others definitions and exercises

Polpetta commented:
@EmanueleC per favore d'ora in poi usa gli spazi e non i tab negli elenchi puntati! 😊 

Infine, quando crei un nuovo file ricordati di dargli sempre un numero (che indica la posizione di dove sara' incluso). Questo e' un requisito per il `make`, altrimenti lo script di compilazione impazzisce! 😁 
Comunque penso che sia meglio che questa cosa sia un glossario dei termini piu' che un generico elenco puntato. Vedo se la conversione e' facile da fare.

Polpetta committed:
Create an appendix for definitions and exercises.
Now at the end of the document there are two new sections:
-Definitions: a list of useful definitions
-Exercises: in this section there are exercises with solutions.
In order to write down the exercises, a package 'exercises' has
been added. Check its documentation here:
http://www.ctan.org/pkg/exercise

Polpetta committed:
Again, no more tabs, only spaces. Remove the \newpage command

Polpetta committed:
Substituted tabs with spaces

Polpetta committed:
Split theory and laboratory in two chapters

Polpetta committed:
Small changes to authors and title

Polpetta committed:
Merge branch 'develop' into general
